### ISSUE TITLE
Switch with button always on and off when pressed

### ISSUE THREAD

**Issue by lmaced0** [KaufHA/kauf-rgb-switch#20]
Hello!!!

First of all, I bought a bunch of switches because I really like the product. Thank you for this, it changed how notifications happens throughout my house.

From the 5 switches I bought, one came DOA, no Wi-Fi, just blinking red light switch was rebooting every 5-7 seconds. That one I'm returning but I have another one that has a weird issue that may be worth saving....

The button status is always On and when pressed it off and when released, it's back on. It's basically inverted.

Here's a video of the issue:
https://youtu.be/xzc8gYiB9Fc

**Comment by veeds2** [comment]
I have the same issue, 

pressing the switch on /off will just revert to the original state.  If the light was on and I pressed the switch, the switch would change color and then revert back to on state without turning off the light.

**Comment by lmaced0** [comment]
Yeah, I'm assuming wires were probably routed incorrectly. I'm returning it.

**Comment by veeds2** [comment]
I think its a software issue, the hardware is solid

**Comment by bkaufx** [comment]
Have you tried re-flashing ESPHome on the switch?  I would do it through ESPHome dashboard and see if it works right.  It kind of looks like a return that someone flashed something weird on and then amazon re-sold the return instead of sending to me.

**Comment by lmaced0** [comment]
On the one flashing red and rebooting, I couldn’t  WiFi wasn’t showing up and I don’t have the tools to flash it through the cable so I didn’t try.

For the one on the video with the inverse button. I tried flashing many times. Did not work.

**Comment by bkaufx** [comment]
ok e-mail me brian@kaufha.com and I'll send you a couple of replacements.
